example/lib/lsp_exploration/lsp/messages/initialize.dart

```diff
@@ -133,8 +133,10 @@ class LspClientCapabilities {
             'documentChanges': true,
             'resourceOperations': ['create', 'rename', 'delete']
           },
+          "applyEdit": true,
```

Comment:
We already have "applyEdit" set above. This is probably a merge conflict with my latest merged PR.